core/lib/workarea/ext/mongoid/embedded_children.rb

```diff
@@ -0,0 +1,25 @@
+module Mongoid
+  module Document
+    #
+    # Returns all the embedded children in this document by recursion.
+    #
+    # *WARNING!*
+    # If you have recursively defined relations, this method will cause a stack
```

Comment:
what does this mean? is it like...

```ruby
class Foo
  include Mongoid::Document

  embeds_many :foos
end
```

When I did that, I got:

```
[1] pry(main)> root = Foo.create!
foo = root.foos.create!
last = foo=> #<Foo _id: 5dc20ae585ed705ecde2d28e, >
[2] pry(main)> foo = root.foos.create!
last = foo=> #<Foo _id: 5dc20ae585ed705ecde2d28f, >
[3] pry(main)> last = foo.foos.create!
=> #<Foo _id: 5dc20ae785ed705ecde2d290, >
[4] pry(main)> last.embedded_children
=> []
[5] pry(main)> foo.embedded_children
=> [#<Foo _id: 5dc20ae785ed705ecde2d290, >]
[6] pry(main)> root.embedded_children
=> [#<Foo _id: 5dc20ae585ed705ecde2d28f, >, #<Foo _id: 5dc20ae785ed705ecde2d290, >
```

no errors. and I couldn't add an `embedded_in :foo` because Mongoid wouldn't let me save since there were no other Foo models in the DB to latch onto.

Reply:
Good call, you're right I'll update.